dependencies.gradle

```diff
@@ -2,8 +2,8 @@
 ext.dep = [
   "assertj": "org.assertj:assertj-core:3.13.2",
   "awaitility": "org.awaitility:awaitility-kotlin:4.0.2",
-  "awsApi": "com.amazonaws:aws-java-sdk:1.11.144",
-  "awsSqs": "com.amazonaws:aws-java-sdk-sqs:1.11.144",
+  "awsApi": "com.amazonaws:aws-java-sdk:1.11.705",
```

Comment:
fyi the last time we bumped this, it had to be rolled back because it caused version problems for projects on vitess. cc: @jjestrel

Reply:
Was it because of the vitess jdbc driver? I think all projects have moved off the driver by now.

Reply:
yes it was the vitess jdbc driver.

```
java.lang.IllegalStateException: Expected to be healthy after starting. The following services are not running: {STARTING=[DataSourceService [NEW], SessionFactoryService [NEW], SchemaMigratorService [NEW], ZkLeaseManager [NEW], ZkClusterWeightsService [STARTING]], FAILED=[PingDatabaseService [FAILED]]}
    	at com.google.common.util.concurrent.ServiceManager$ServiceManagerState.checkHealthy(ServiceManager.java:773)
    	at com.google.common.util.concurrent.ServiceManager$ServiceManagerState.awaitHealthy(ServiceManager.java:585)
    	at com.google.common.util.concurrent.ServiceManager.awaitHealthy(ServiceManager.java:316)
    	at misk.MiskApplication.startServiceAndAwaitTermination(MiskApplication.kt:94)
    	at misk.MiskApplication.doRun$misk(MiskApplication.kt:51)
    	at misk.MiskApplication.run(MiskApplication.kt:35)
    	at com.squareup.plasma.service.PlasmaServiceKt.main(PlasmaService.kt:19)
    	Suppressed: com.google.common.util.concurrent.ServiceManager$FailedService: PingDatabaseService [FAILED]
    	Caused by: java.lang.NoClassDefFoundError: io/netty/internal/tcnative/SSLPrivateKeyMethod
    		at io.netty.handler.ssl.SslContext.newClientContextInternal(SslContext.java:827)
    		at io.netty.handler.ssl.SslContextBuilder.build(SslContextBuilder.java:576)
    		at io.vitess.client.grpc.GrpcClientFactory.createTls(GrpcClientFactory.java:149)
    		at io.vitess.jdbc.VitessVTGateManager.getVtGateConn(VitessVTGateManager.java:213)
    		at io.vitess.jdbc.VitessVTGateManager.updateVtGateConnHashMap(VitessVTGateManager.java:131)
    		at io.vitess.jdbc.VitessVTGateManager.access$300(VitessVTGateManager.java:43)
    		at io.vitess.jdbc.VitessVTGateManager$VTGateConnections.<init>(VitessVTGateManager.java:72)
    		at io.vitess.jdbc.VitessConnection.connect(VitessConnection.java:97)
    		at io.vitess.jdbc.VitessDriver.connect(VitessDriver.java:60)
    		at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:121)
    		at misk.jdbc.PingDatabaseService.connect(PingDatabaseService.kt:69)
    		at misk.jdbc.PingDatabaseService.connectToDataSource(PingDatabaseService.kt:48)
    		at misk.jdbc.PingDatabaseService.access$connectToDataSource(PingDatabaseService.kt:21)
    		at misk.jdbc.PingDatabaseService$startUp$1.invoke(PingDatabaseService.kt:31)
    		at misk.jdbc.PingDatabaseService$startUp$1.invoke(PingDatabaseService.kt:21)
    		at misk.backoff.RetriesKt.retry(Retries.kt:17)
    		at misk.jdbc.PingDatabaseService.startUp(PingDatabaseService.kt:29)
    		at com.google.common.util.concurrent.AbstractIdleService$DelegateService$1.run(AbstractIdleService.java:62)
    		at com.google.common.util.concurrent.Callables$4.run(Callables.java:119)
    		at java.base/java.lang.Thread.run(Thread.java:834)
    	Caused by: java.lang.ClassNotFoundException: io.netty.internal.tcnative.SSLPrivateKeyMethod
    		at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:583)
    		at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
    		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
    		... 20 common frames omitted```